pages/dkp/kommander/2.3/release-notes/2.2.0/index.md

```diff
@@ -21,15 +21,15 @@ beta: false
 
 Welcome to D2iQ Kubernetes Platform (DKP) 2.3! This release provides new features and enhancements to improve the user experience, fix reported issues, integrate changes from previous releases, and maintain compatibility and support for other packages used in Konvoy.
 
-DKP 2.3 supports Kubernetes versions between xxxx and xxxx. Any cluster you want to attach using DKP 2.3 must be running a Kubernetes version in this range.
+DKP 2.3 supports Kubernetes versions between 1.21.0 and 1.23.x. Any cluster you want to attach using DKP 2.3 must be running a Kubernetes version in this range.
```

Comment:
I think this page is in the wrong directory, the path is: `pages/dkp/kommander/2.3/release-notes/2.2.0/index.md`

Shouldn't it be in the 2.3.0 directory?

Reply:
You are right! The name of the folder needs to change. However, I decided to do that in another PR to focus on Kubernetes in this one. (When you change the name of a folder, the whole topic appears as new, even if it is not).